packages/Card/src/components/Footer/Footer.styles.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
+import { spacer } from "@paprika/stylers/lib/helpers";
+
+export const footerStyles = styled.div`
+  background: ${tokens.color.blackLighten70};
+  border-bottom-left-radius: ${tokens.card.borderRadius};
```

Comment:
perhaps set the border radius here the the same as you did at https://github.com/acl-services/paprika/pull/531/files#diff-a13b744a53b71e448868586e137c4388R5